### PR TITLE
feat: (DSO-2128) Add envi command to create environments manually

### PR DESCRIPTION
**Pull request by javiercm1410**

### CODE REVIEW

File: pkg/providers/manual/manual.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+type ManualOptions struct {
+	Value []string
+	Path  []string
+}
+
+func GenerateEnvFile(options ManualOptions) error {
+	envData := make(parser.EnvVarObject)
+	for _, value := range options.Value {
+		kp := strings.SplitN(value, "=", 2)
```
Comment (angelmadames) [inline pkg/providers/manual/manual.go:18]: Let's avoid short variable names.

File: cmd/manual.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	return manual.ManualOptions{
+		Value: value,
+		Path:  path,
+	}
+}
+
+func init() {
+	RootCmd.AddCommand(ManualCommand)
+
+	ManualCommand.PersistentFlags().StringArrayP("value", "v", []string{}, "Manual values to add")
```
Comment (angelmadames) [inline cmd/manual.go:38]:
```suggestion
	ManualCommand.PersistentFlags().StringArrayP(
	  "value",
	  "v",
	  []string{},
	  "Manual values to add",
)
```

**Review by b-nicasio — APPROVED** [review]

File: cmd/manual.go [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+	// Required flags
+	if err := ManualCommand.MarkPersistentFlagRequired("value"); err != nil {
+		log.Fatalf("Error marking 'value' flag as required: %v", err)
```
Comment (angelmadames) [inline cmd/manual.go:42]:
```suggestion
		log.Fatalf(
		    "Error marking 'value' flag as required: %v",
		    err
		)
```

**Review by angelmadames — CHANGES_REQUESTED** [review]

File: pkg/providers/manual/manual.go (hunk 1 above)
Comment (javiercm1410) [reply pkg/providers/manual/manual.go]: var updated

File: cmd/manual.go (hunk 3 above)
Comment (javiercm1410) [reply cmd/manual.go]: fixed

File: cmd/manual.go (hunk 2 above)
Comment (javiercm1410) [reply cmd/manual.go]: fixed

**Review by angelmadames — APPROVED** [review]

**Pull request merged**